polygon_coverage_solvers/include/polygon_coverage_solvers/impl/graph_base_impl.h

```diff
@@ -334,7 +334,7 @@ GraphBase<NodeProperty, EdgeProperty>::getAdjacencyMatrix() const {
       if (edgeExists(edge) && getEdgeCost(edge, &cost)) {
         m[i][j] = doubleToMilliInt(cost);
       } else {
-        m[i][j] = std::numeric_limits<int>::max();
+        m[i][j] = std::numeric_limits<int>::max(); // TODO: overflow!
```

Comment:
@rikba You might want to have a look at this line. Setting the weights to `std::numeric_limits<int>::max()` causes an integer overflow within GLKH. What value should we use instead?

Reply:
If I remember correctly `std::numeric_limits<int>::max()`was used in gkma to note edges in the adjacency matrix that are not traversable. I had a look into the GTSPLIB instances in GLKH-1.0/GTSPLIB and they have different large values there, e.g., 10ftv47.gtsp has `100000000`, 4br17.gtsp has `9999`, 11ft53.gtsp has `9999999`. So maybe it's ok to allocate `9999999` like you do below but add a TODO that we want to setup the problem in [EDGE_DATA_FORMAT](https://github.com/crigroup/lkh/blob/master/glkh_solver/src/ReadProblem.c#L91-L95) and let GKLH handle the disconnected edges.